rdrf/rdrf/forms/components.py

```diff
@@ -135,10 +135,19 @@ def _get_template_data(self):
             "consent_locked": self.consent_locked,
             "clinician_form_link": self._get_clinician_form_link(),
             "proms_link": self._get_proms_link(),
+            "allow_proms_add": self._get_proms_add(),
         }
 
         return data
 
+    def _get_proms_add(self):
+        metadata = self.registry_model.metadata
+        allow_proms_add = metadata.get("allow_proms_add", None)
+        if allow_proms_add is None:
+            return ""
+        else:
+            return allow_proms_add in ['True', 'true', '1', 'yes', 'Yes']
```

Comment:
Just specify that we use true or false

You could also just use a string in the features list and check registry_model.has_feature

features: [ ... ,  "can_add_proms"]


reg.has_feature("can_add_proms")


pref stick to this pattern